mod_str

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 11 17:16:56 2019
+
+@author: paridhikumar
+"""
+class mod_str():
+    """ class asks user for string input and operation they want to perform on string.the operation is performed.
+    --Attributes--
+    message: the message the user inputs
+    op: the operation the user wants performed
+    """
+    message: str
+    op: str
+
+    def __init__(self) -> None:
+        a = str(input("Input your message: "))
+        self.message = a
+        b = str(input("What operation do you want to perform?: ")) ; print("Operations:","- length (returns lenght of message)", "- is uppercase","  (checks if all all alphabets in a message are uppercase alphabets.If the message contains at least one uppercase alphabet, it returns False.)","- uppercase (returns all alphabets in uppercase)","- is lowercase","  (checks if all all alphabets in a message are lowercase alphabets.If the message contains at least one uppercase alphabet, it returns False.)","- lowercase (returns all alphabets in lowercase)","- capitalize (converts first character to Capital Letter),"- title case (returns a title cased string)","- all alphabets (checks if all characters are alphabets)","- all numbers (checks if all characters in a string are numeric characters)","- all space (checks if there are only whitespace characters in the string)",sep="\n")
```

Comment:
Rather having a info can we make it more user friendly ? 

Can we display a list in console from where user can select exact operation needs to be performed !